framework/src/main/kotlin/br/com/zup/beagle/widget/ui/GridView.kt

```diff
@@ -19,8 +19,10 @@ package br.com.zup.beagle.widget.ui
 import br.com.zup.beagle.widget.Widget
 import br.com.zup.beagle.widget.action.Action
 import br.com.zup.beagle.widget.context.Bind
+import br.com.zup.beagle.widget.context.Context
 import br.com.zup.beagle.widget.context.ContextComponent
-import br.com.zup.beagle.widget.context.ContextData
+import br.com.zup.beagle.widget.ui.GridViewDirection.HORIZONTAL
```

Comment:
<table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-backend-kotlin/blob/1767037c5777b4b44c2b78a548a8a8a8514b5d6b/framework/src/main/kotlin/br/com/zup/beagle/widget/ui/GridView.kt#L24"></span>The import 'br.com.zup.beagle.widget.ui.GridViewDirection.HORIZONTAL' is unused.</td>
    </tr>
  </tbody>
</table>